resources/php_agent.rb

```diff
@@ -28,7 +28,6 @@
 
 attribute :mode,           kind_of: [Integer, String], default: 0o644
 
-attribute :cookbook,       kind_of: String, default: 'newrelic-ng'
```

Comment:
I’d argue the other way: there are plenty of reasons an operator might supply their own `newrelic.ini.php`; I’ve used this feature myself when we still used this cookbook for our New Relic integration.